apps/new/widget/page/projects/Editor.jsx

```diff
@@ -271,6 +271,40 @@ const handleTags = (tags) => {
   setTags(filtered);
 };
 
+// Handle NEAR Catalog Projects
+const catalogProjectId = props.catalogProjectId;
+const catalogProjectData = null;
+if (catalogProjectId) {
+  const indexer = "https://nearcatalog.xyz/wp-json/nearcatalog/v1";
```

Comment:
I think we use it in multiple places, can we move it to replacements?

Reply:
Care to elaborate?